
#include "centre_player_handler.h"

///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE



void CentrePlayerUtilityHandler::SendTipToClient(entt::entity player,const ::TipInfoMessage* request,
	::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE


}



void CentrePlayerUtilityHandler::KickPlayer(entt::entity player,const ::CentreKickPlayerRequest* request,
	::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE


}


